src/js/NavAccordion.js

```diff
@@ -1,54 +1,71 @@
 
-			var $NavAccordion_Item = $(this);
-			var $subNav = $NavAccordion_Item.children( 'ul' );
+			var subNav = Array.prototype.filter.call( navAccordion_Item.children, function( el ) {
+				return el.tagName === 'UL';
+			} );
 
-			if ( $subNav.length > 0 ) {
+			if ( subNav.length < 1 ) {
+				return;
+			} else {
+				subNav = subNav[0];
+			}
+
+			var openHeight = subNav.offsetHeight;
+			var style      = getComputedStyle( subNav );
 
-				var openHeight           = $subNav.outerHeight();
-				var $NavAccordion_Toggle = $( '<button/>', {
-					text: 'Toggle',
-					class: 'Btn NavAccordion_Toggle'
-				} );
+			openHeight += parseInt( style.marginTop ) + parseInt( style.marginBottom );
 
-				var toggleSubNav = function( show ) {
+			var navAccordion_Toggle = document.createElement( 'BUTTON' );
+			navAccordion_Toggle.appendChild( document.createTextNode( "Toggle" ) );
+			navAccordion_Toggle.classList.add( 'Btn' );
+			navAccordion_Toggle.classList.add( 'NavAccordion_Toggle' );
 
-					if ( 'undefined' === typeof( show ) ) {
-						show = $NavAccordion_Item.hasClass( 'NavAccordion_Item-Closed' );
-					}
+			var anchor = Array.prototype.filter.call( navAccordion_Item.children, function( el ) {
+				return el.tagName === 'A';
+			} );
 
-					if ( show ) {
-						$NavAccordion_Item.removeClass( 'NavAccordion_Item-Closed' );
-						$NavAccordion_Item.addClass( 'NavAccordion_Item-Open' );
-						$subNav.css( 'max-height', openHeight );
-					} else {
-						$NavAccordion_Item.addClass( 'NavAccordion_Item-Closed' );
-						$NavAccordion_Item.removeClass( 'NavAccordion_Item-Open' );
-						$subNav.css( 'max-height', 0 );
-					}
+			if ( anchor.length ) {
+				anchor[0].appendChild( navAccordion_Toggle );
+			}
 
+			var toggleSubNav = function( show ) {
+
+				if ( 'undefined' === typeof( show ) ) {
+					show = navAccordion_Item.classList.contains( 'NavAccordion_Item-Closed' );
 				}
 
-				$NavAccordion_Toggle.appendTo( $NavAccordion_Item.children('.NavAccordion_Anchor') );
-				toggleSubNav( false );
+				if ( show ) {
+					navAccordion_Item.classList.remove( 'NavAccordion_Item-Closed' );
+					navAccordion_Item.classList.add( 'NavAccordion_Item-Open' );
+					subNav.style['max-height'] = openHeight + 'px';
+				} else {
+					navAccordion_Item.classList.add( 'NavAccordion_Item-Closed' );
+					navAccordion_Item.classList.remove( 'NavAccordion_Item-Open' );
+					subNav.style['max-height'] = 0;
+				}
 
-				$NavAccordion_Toggle.click( function( e ) {
-					e.preventDefault();
-					$(this).blur();
-					toggleSubNav();
-				});
 			}
 
+			toggleSubNav( false );
+
+			navAccordion_Toggle.addEventListener( 'click', function( event ) {
+				event.preventDefault();
+				navAccordion_Toggle.blur();
+				toggleSubNav();
+			});
+
 		} );
 
 	}
 
-	$('.NavAccordion').each( function() {
-		navAccordion( $(this) );
-	} );
+	var els = document.getElementsByClassName('NavAccordion');
```

Comment:
I've added a check to the top of the file that will simply return if any of these methods are NOT supported.